specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,165 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../document-status.md)
+
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Page View](#page-view)
+- [Ajax](#ajax)
+- [Exception](#exception)
+- [UserAction](#useraction)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [AjaxTiming](#ajaxtiming)
+- [ResourceTiming](#resourcetiming)
+
+<!-- tocstop -->
+
+</details>
+
+
+This document describes the semantic conventions for browser events. The events may be represented either as Span Events or Log Events.
+
+All events have the following three high-level attributes. The domain has a fixed value of `browser`. The event name is specified at the beginning of each section below. The payload of the event goes in an attribute called `event.data` and the fields described below for each event go into this attribute.
+
+| Key  | Value | Description  |
+|---|---|---|
+| `event.domain` | browser | Fixed value |
+| `event.name` | | Described in each section below|
+| `event.data` | | A map of key/value pairs, with the keys for each event described in following sections|
+
+
+
+## Page View
+
+<!-- semconv browser.page_view -->
+The event name MUST be `page_view`.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `referrer` | string | Referrer URL for the browser page (`document.referrer`) | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Browser page type | `0` | Required |
+| `user_consent` | boolean | Whether or not user provided consent on the page (consent for what?) |  | Recommended |
+| [`http.url`](../../../trace/semantic_conventions/http.md) | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. [1] | `https://en.wikipedia.org/wiki/Main_Page`; `https://en.wikipedia.org/wiki/Main_Page#foo` | Required |
```

Comment:
I'm confused why you decided to pluck out this field (which is actually defined in varying attribute as browser.page.url) and add it here!